src/api/util.js

```diff
@@ -1339,7 +1339,7 @@ export function fromAssetUnit(unit) {
   const label = fromLabel(unit.slice(56, 64));
   const name = (() => {
     const hexName = Number.isInteger(label) ? unit.slice(64) : unit.slice(56);
-    return hexName || null;
+    return unit.length === 56 ? "" : hexName || null;
```

Comment:
I'm not sure in what situations it might be correct for a token name to be `null` or why this expression is formulated this way, but in any such case the asset will currently produce an error in the wallet. I added the empty token name as a special case in consideration of such situations, but if these cases exist legitimately it seems like the `null` should still be handled elsewhere.